### PR TITLE
Added three new Atomic tools

### PR DESCRIPTION
**Pull request by thelonestander**
Jackhammer, Drill, and Auger

#### Purpose of change

I love atomic gear in game, we have atomic phones, atomic robots, next up high end atomic tools that the military might have used to create the many outposts and helipads we find? Currently in game if you have a static base you will eventually run out of batteries, there are other power options as well such as electric. These tools are to provide late game, possibilities to expand bases, or possibly expand a current overmap location you've selected as a base, and not have to track down batteries to make them work. 

#### Describe the solution

I added the three new atomic tools as explained above. 

#### Describe alternatives you've considered

- Not adding them
- Figuring out other tools to make atomic
- Revert to the stone age. 

#### Testing

Added to the game, and tested on a new character. Spawned in the new tools, they work! 

#### Additional context

Screenshot (553)
Screenshot (554)

### CODE REVIEW

**Comment by DrPariah** [comment]
Huh, nifty. Been thinking about this one for a while. I would've added an Atomic Chainsaw with Atomic Combat Chainsaw as well, but that's just me.

**Comment by Coolthulhu** [comment]
Build error is about missing double space in jsons. Check the details in the failing test.

File: data/json/items/tool/workshop.json [hunk 1]
```diff
@@ -1180,5 +1180,63 @@
     "color": "light_gray",
     "looks_like": "hand_drill",
     "qualities": [ [ "DRILL", 1 ] ]
+  },
+  {
+    "id": "Atomic_Jackhammer",
+    "type": "TOOL",
+    "category": "tools",
+    "name": { "str": "atomic jackhammer" },
+    "description": "This is a construction tool for drilling through hard rock or other surfaces.  This one runs on the power of the atom!  Has a feint green glow",
```
Comment (Coolthulhu) [inline data/json/items/tool/workshop.json:1189]: Faint glow. Feint is when you fake an attack.

**Comment by Coolthulhu** [comment]
The tools will need some spawn or a recipe.
I don't see a good place for a spawn, since they are mundane tools, just usable forever.
As for the recipe, we'd need to figure out how much use does a typical battery-powered tool of the type get to find how costly should the tool be.

I recall some recipes explicitly calling for certain tools. You'll have to check which recipes use `"cordless_drill"` specifically, then do the same for auger and jackhammer.
If there are many recipes like this, the atomic/non-atomic variants will have to be wrapped in a `"requirement"`. Otherwise it's OK to just modify the recipes to add atomic version as another option.

File: data/json/items/tool/workshop.json [hunk 2]
```diff
@@ -1180,5 +1180,63 @@
...
+    "weight": "40000 g",
+    "looks_like": "jackhammer",
```
[the remark this answers is not in the document]
Comment (chaosvolt) [reply data/json/items/tool/workshop.json:1191]: Only issue I can see with that is I'm not sure how one would un-set an ammo type.

**Comment by KheirFerrum** [comment]
It's worth noting that in game, our usual atomic aligned non-charge consuming tools are the non-power hungry tools like lights, coffee makers, etc. This is meant to portray how the power draw to power source is so negligible that it is _functionally_ unlimited.

Jackhammers, Augurs and drills are incredibly power hungry, and making them use no charges on the basis of being plutonium powered starts to stretch the consistency of the setting. I'd rather have a battery modification that causes it to allow the use of plutonium cells, which it will eat through decently quickly.

I'm also not really sure about the battery thing being an actual issue, since you will quickly amass a large number of rechargeable heavy batteries that a single solar panel with recharging station will fill faster than you can use.

**Comment by thelonestander** [comment]
> It's worth noting that in game, our usual atomic aligned non-charge consuming tools are the non-power hungry tools like lights, coffee makers, etc. This is meant to portray how the power draw to power source is so negligible that it is _functionally_ unlimited.
> 
> Jackhammers, Augurs and drills are incredibly power hungry, and making them use no charges on the basis of being plutonium powered starts to stretch the consistency of the setting. I'd rather have a battery modification that causes it to allow the use of plutonium cells, which it will eat through decently quickly based on the power_draw...
> 
> I'm also not really sure about the battery thing being an actual issue, since you will quickly amass a large number of rechargeable heavy batteries that a single solar panel with recharging station will fill faster than you can use.

Your not wrong, this PR was mainly to see how you felt about them in game. I enjoy atomic items, and usually collected them. Not sure why. If it doesn't fit I can close it if you see fit. For the batteries, I was about three years into the game, and I managed to run out of batteries. Not everyone will, but I did.

**Pull request closed without merging**